pkg/reconciler/serverlessservice/resources/services_test.go

```diff
@@ -211,6 +226,11 @@ func TestMakePublicService(t *testing.T) {
 				Protocol:   corev1.ProtocolTCP,
 				Port:       pkgnet.ServiceHTTP2Port,
 				TargetPort: intstr.FromInt(networking.BackendHTTP2Port),
+			}, {
+				Name:       pkgnet.ServicePortNameHTTPS,
+				Protocol:   corev1.ProtocolTCP,
+				Port:       pkgnet.ServiceHTTPSPort,
+				TargetPort: intstr.FromInt(networking.BackendHTTPSPort),
```

Comment:
Since we are touch `filterSubsetPorts` could you enhance TestFilterSubsetPorts with related cases?

Reply:
Thank you. Done.